sct.py

```diff
@@ -487,85 +486,82 @@ def list_resources(ctx, user, test_id, get_all, get_all_running, verbose):
-              default='eu-west-1',
-              help="a region to look for AMIs (default: eu-west-1)")
-@click.option('-a', '--arch',
-              type=click.Choice(AwsArchType.__args__),
-              default='x86_64',
-              help="architecture of the AMI (default: x86_64)")
-@click.argument('version', type=str, default='branch-3.1:all')
-def list_ami_branch(region: str, arch: AwsArchType, version: str):
-    add_file_logger()
-
-    def get_tags(ami):
-        return {i['Key']: i['Value'] for i in ami.tags}
-
-    if ":" not in version:
-        version += ":all"
-
-    tbl = PrettyTable(field_names=["Name", "ImageId", "CreationDate", "BuildId", "Test Status"], align="l")
-    for ami in get_branched_ami(scylla_version=version, region_name=region, arch=arch):
-        tags = get_tags(ami)
-        test_status = [(k, v) for k, v in tags.items() if k.startswith('JOB:')]
-        test_status = [click.style(k, fg='green') for k, v in test_status if v == 'PASSED'] + \
-                      [click.style(k, fg='red') for k, v in test_status if not v == 'PASSED']
-        test_status = ", ".join(test_status) if test_status else click.style('Unknown', fg='yellow')
-        tbl.add_row([ami.name, ami.image_id, ami.creation_date,
-                     tags.get('build-id', tags.get('build_id')), test_status])
-    click.echo(tbl.get_string(title="Scylla AMI branch versions"))
-
-
-@cli.command("list-gce-images-versions", help="list Scylla formal GCE images versions")
-def list_gce_images_versions():
-    add_file_logger()
-
-    tbl = PrettyTable(field_names=["Name", "ImageId", "CreationDate"], align="l")
-    for image in get_scylla_gce_images_versions():
-        tbl.add_row([image.name, image.extra["selfLink"], image.extra["creationTimestamp"]])
-    click.echo(tbl.get_string(title="Scylla GCE images versions"))
-
-
-@cli.command("list-gce-images-branch", help="""list Scylla branched GCE images versions
-    \n\n[VERSION] is a branch version to look for, ex. 'branch-2019.1:latest', 'branch-3.1:all'""")
-@click.argument("version", type=str, default="branch-3.1:all")
-def list_gce_images_branch(version):
-    add_file_logger()
-
-    if ":" not in version:
-        version += ":all"
-
-    tbl = PrettyTable(field_names=["Name", "ImageId", "CreationDate", "BuildId", "Test Status"], align="l")
-    for image in get_branched_gce_images(scylla_version=version):
-        tbl.add_row([
-            image.name,
-            image.extra["selfLink"],
-            image.extra["creationTimestamp"],
-            image.extra["labels"].get("build-id") or image.name.rsplit("-build-", maxsplit=1)[-1],
-            click.style("Unknown", fg="yellow"),
-        ])
-    click.echo(tbl.get_string(title="Scylla GCE images branch versions"))
+    branch_fields = ["Backend", "Name", "ImageId", "CreationDate", "BuildId", "Arch", "ScyllaVersion"]
+    version_fields = ["Backend", "Name", "ImageId", "CreationDate"]
```

Comment:
shouldn't it have also `Arch` and `BuildId` in here?

Reply:
FYI, this is a bug, I've raised it with scylla-pkg:
https://github.com/scylladb/scylla-cluster-tests/issues/5041

we should have labels across all images.

Reply:
> FYI, this is a bug, I've raised it with scylla-pkg: #5041
> 
> we should have labels across all images.

@fruch , this issue 5041 isn't an issue to scylla-pkg... 

@KnifeyMoloko , please add a comment to the `version_fields` mentioning the issue @fruch mentioned (https://github.com/scylladb/scylla-pkg/issues/2995) so we will know that we can expand it once the issue is fixed

Reply:
Well the bug I've paste is kind of related, if we'll want azure support :)

Reply:
TODO comment added. I also created a task in Trello, so we don't loose sight of this one (https://trello.com/c/UoVmGEmw)